torch/csrc/distributed/c10d/TraceUtils.h

```diff
@@ -8,8 +8,15 @@
 #include <torch/csrc/jit/serialization/pickler.h>
 #include <torch/csrc/profiler/combined_traceback.h>
 
+#include <fmt/compile.h>
```

Comment:
Hmmm with this PR, if I later include TraceUtils.h into gloo, I got compile error:

pytorch/torch/csrc/distributed/c10d/TraceUtils.h:11:10: fatal error: fmt/compile.h: No such file or directory
   11 | #include <fmt/compile.h>
       |          ^~~~~~~~~~~~~~~

Reply:
@Skylion007 Do you have any idea how to forward fix it? Shall I change more CMakeList.txt?